contracts/abstracts/ListingManager.sol

```diff
@@ -3,7 +3,9 @@ pragma solidity ^0.8.19;
 
 import './Controlable.sol';
 
-abstract contract ListingManager is Controlable {
+import '@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol';
+
+abstract contract ListingManager is Controlable, IERC721Receiver {
```

Comment:
Retire IERC721Receiver de la liste l'inéritance, nous n'avons pas besoin d'inériter une interface